enterprise/j2ee.earproject/src/org/netbeans/modules/j2ee/earproject/EarProjectGenerator.java

```diff
@@ -543,8 +543,7 @@ private AntProjectHelper setupProject() throws IOException {
             srcLevel = v.toString();
             // #89131: these levels are not actually distinct from 1.5.
             // #181215: JDK 6 should be the default source/binary format for Java EE 6 projects
-            if (srcLevel.equals("1.7"))
-                srcLevel = "1.6";
```

Comment:
This is several places - I saw a source level defined in one of the classes touched by this PR, why is that not used?